overlord/overlord.go

```diff
@@ -56,6 +57,10 @@ import (
 	"github.com/snapcore/snapd/timings"
 )
 
+const (
+	overlordStateLockTimeout = 20 * time.Second
```

Comment:
I wonder if this is enough on all slow devices

Reply:
Bumped it to a minute